uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/my-leave/app.js

```diff
@@ -8,7 +8,7 @@ define([
   'common/services/angular-date/date-format',
   'leave-absences/shared/ui-router',
   'leave-absences/my-leave/modules/config',
-  'leave-absences/my-leave/components/my-leave',
+  'leave-absences/my-leave/components/my-leave-container',
```

Comment:
> manager-leave and my-leave components renamed to manager-leave-container and my-leave-container to fix the JS error when Debugging is turned on.

but why this error happens in the first place? why does it matter what a component is called? Also, aren't you changing the name of the file without changing the name of the component?